src/GameLogic.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace GOL;
+
+use GOL\Boards\Board;
+use GOL\Boards\History;
+use GOL\Rules\Rule;
+
+/**
+ * Controls the game logic of the game.
+ * Use nextGeneration to calculate the next generation.

```

Comment:
If you use nextGeneration() (with parantheses) then doxygen will automatically turn it into a link pointing to the method documentation.